photos.html

```diff
@@ -1,111 +1,203 @@
         </div>
-      </header>
+      </section>
 
-      <main class="main-content">
-        <section class="preamble">
-          <div class="preamble__inner">
-            <h2 class="preamble__header">Photo and video</h2>
-            <p class="preamble__paragraph">Can't decide to travel because of the right time? We hope photos will help you forget about politics and economics.</p>
+      <div class="gallery">
+        <section class="gallery__item">
+          <div class="gallery__inner">
+            <div class="gallery__picture">
+              <picture>
+                <source media="(max-width: 640px)" srcset="img/photo-sunset-mobile@1x.jpg 1x, img/photo-sunset-mobile@2x.jpg 2x">
+                <source media="(max-width: 1000px)" srcset="img/photo-sunset-tablet@1x.jpg 1x, img/photo-sunset-tablet@2x.jpg 2x">
+                <source media="(min-width: 1001px)" srcset="img/photo-sunset-desktop@1x.jpg 1x, img/photo-sunset-desktop@2x.jpg 2x">
+                <img src="img/photo-sunset-mobile@1x.jpg" alt="Sunset">
+              </picture>
+            </div>
+            <div class="gallery__caption">
+              <h3 class="gallery__title">Outback</h3>
+              <p class="gallery__author">Photo by: Boris</p>
+              <p class="gallery__likes gallery__likes--icon">1350</p>
+            </div>
           </div>
         </section>
 
-        <div class="gallery">
+        <section class="gallery__collection">
           <section class="gallery__item">
-            <a class="gallery__image galley__image--main"></a>
             <div class="gallery__inner">
-              <h3 class="gallery__title">Outback</h3>
-              <p class="gallery__author">Photo by: Boris</p>
-              <p class="gallery__likes">1350</p>
-            </div>
-          </section>
 
-          <section class="gallery__collection">
-            <section class="gallery__item">
-              <a class="gallery__image galley__image--vegetation"></a>
-              <div class="gallery__inner">
+              <div class="gallery__picture">
+                <picture>
+                  <source media="(max-width: 640px)" srcset="img/photo-cactus-mobile@1x.jpg 1x, img/photo-cactus-mobile@2x.jpg 2x">
+                  <source media="(max-width: 1000px)" srcset="img/photo-cactus-tablet@1x.jpg 1x, img/photo-cactus-tablet@2x.jpg 2x">
+                  <source media="(min-width: 1001px)" srcset="img/photo-cactus-desktop@1x.jpg 1x, img/photo-cactus-desktop@2x.jpg 2x">
+                  <img src="/img/photo-cactus-mobile@1x.jpg" alt="Local vegetation: Cactus">
+                </picture>
+              </div>
+
+              <div class="gallery__caption">
                 <h3 class="gallery__title">Local vegetation</h3>
                 <p class="gallery__author">Photo by: Sergey</p>
-                <p class="gallery__likes">143</p>
+                <p class="gallery__likes gallery__likes--icon">143</p>
```

Comment:
unnecessary styling class `gallery__likes--icon` ?

Reply:
I think like should be a button, it's not a decoration. 

![image](https://user-images.githubusercontent.com/1570578/35457857-3e95061c-02d2-11e8-879d-024f428bdb8d.png)

Reply:
semantically I think button and number of likes should in one container. It could be an independent block event. so absolute positioning won't work here